src/antares/service/base_services.py

```diff
@@ -391,6 +391,11 @@ def add_constraint_terms(self, constraint: BindingConstraint, terms: List[Constr
         """
         pass
 
+    @property
+    @abstractmethod
+    def binding_constraints(self) -> dict[str, BindingConstraint]:
+        pass
```

Comment:
Is it normal ?

Reply:
I'm sorry I don't understand this comment, could you clarify?

Reply:
I was refering to the `pass`, shouldn't you do something here. Or if it's normal update the expected response model

Reply:
I'm sorry, still confused. This is the abstract base service, the `pass` is pervasive throughout not just in this method. The response model is for inheritance.

Reply:
Ok nevermind I misunderstood. But I believe you should remove the @property as it's clearly not one.